app/main.py

```
@@ -1,47 +1,22 @@
-    JE_API_ROOT_PATH = f"/{JE_API_ROOT_PATH}"
-
+from .config import (
+    JE_CMS_API_BASE_URL,
+    JE_CMS_API_TOKEN,
+    JE_WEB_BASE_URL,
+    JE_API_ROOT_PATH,
+    JE_API_CORS_ORIGINS,
+    JE_API_CORS_ORIGINS_REGEX,
+)
+from .dependencies import parse_cors_origins
+from .routers import blog, calendar, copyright, health
```

Ruff (0.15.21)

[error] Import `copyright` is shadowing a Python builtin

(A004)